docs/authentication/social-connections/gitlab.mdx

```diff
@@ -27,10 +27,10 @@ First, you need to register a new OAuth GitLab app. Follow the official GitLab i
 
 ![Creating an OAuth GitLab app](/docs/images/authentication-providers/gitlab/7e67b1cb88c1df509cac722d6a6b2d7c624686f8-1919x840.png)
 
-You need to add a name for your new application and the **Redirect URI**. In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable GitLab. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Go back to the GitLab panel, paste the value into the **Redirect URI**, select any scopes that would you like your users to provide and save the application.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **GitLab**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Go back to the GitLab panel, paste the value into the **Redirect URI**, select any scopes that would you like your users to provide and save the application.
 
 Once creation is complete, you'll get redirected to application's panel. Copy the **Application ID** and **Secret.** Go back to the Clerk Dashboard and paste them into the respective fields.
 
 ![Obtaining the Application ID and Client secret](/docs/images/authentication-providers/gitlab/63d7ad6409b75e250377b4d1e9b792dbcecadae1-1920x841.png)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with GitLab is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with GitLab is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with GitLab is now configured for your instance.
```